### ISSUE TITLE
mbsubmit: Allow submitting release after initial import

### ISSUE THREAD

**Issue by arsaboo** [beetbox/beets#5032]
Thanks @doronbehar, for the recent additions to the mbsubmit plugin #4807. Currently, the plugin only allows submission only on initial import. We should also extend the `mbsubmit` command to submit the release after the initial import. 

Specifically, in addition to the track listing, `beet mbsubmit album:"XYZ"` should also give an option to `Open files with Picard`.

**Comment by doronbehar** [comment]
> Specifically, in addition to the track listing, `beet mbsubmit album:"XYZ"` should also give an option to `Open files with Picard`.

Just to make sure I understand, are you describing a scenario that the user has imported an album's folder "as is" or via a spotify / discogs release (via the relevant plugins) and then they wish to submit it to MusicBrainz?

**Comment by arsaboo** [comment]
That is correct...the album is already imported to the beets library with any of the tag sources (e.g., Spotify) and needs to be submitted to MB.

**Comment by Kyubai** [comment]
This would help me a lot.
Currently mbsubmit is using `before_choose_candidate_listener`, but it would have to trigger after the candidates (spotify, deezer, etc. match) has been chosen.


Here's an example:

![image](https://github.com/beetbox/beets/assets/34755366/f551b548-c125-49a0-b4ae-602adcc69b34)

As you can see spotify provides the correct album info, but pressing `p` only prints the information from my local file.
Once I choose `Apply` beets will move on the the next track and I can't print the correct metadata anymore

**Comment by doronbehar** [comment]
@Kyubai are you using the latest unreleased beets? With https://github.com/beetbox/beets/pull/4807 included? I'm not sure I understand what you are trying to say about the event `before_choose_candidate_listener` - The menu you showed above should I think give you the prompt choice to open the files with Picard.
